### DISCUSSION TITLE
Chakra UI for Mobile App (React Native)

### DISCUSSION THREAD

**Discussion by davidcodes17** [chakra-ui/chakra-ui#8976] [Ideas]
Please is there a way you guys can have chakra ui for mobile apps like for react native? if yes please let it be implemented i'm tired of all these lib i've been using

**Comment by vanenshi** [comment]
I can totally relate—I've tried almost every library available, and each one seems to have a few limitations that make it hard to adopt fully. I was even considering building something similar myself! But if the Chakra UI (@segunadebayo) sage team is interested in pursuing this, I’d be more than happy to lend a hand.

**Comment by segunadebayo** [comment]
We have no plans to work on this. Sorry
